### DISCUSSION TITLE
Beş taş sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#11] [Zeka Sorusu] [answered]
Ahmet ile Betül bir oyun oynuyorlar ve oyuna Ahmet başlıyor. Başlangıçta yerde 2018 tane taş vardır. Sırası gelen yerden en az 1 en çok 5 taş alıyor. Son taşı alan oyunu kaybettiğine göre Ahmet ilk hamlede yerden kaç taş alırsa kazanmayı garantiler?

**Comment by yigidi** [comment, marked as answer]
Ahmet, ilk hamlesinde 2018 sayısını (1+5+1+5+...) şeklindeki toplamlardan birini oluşturacak şekilde yerden taş almalıdır.
Betül, her ne taş sayısıyla başlarsa başlasın, Ahmet'in belirlediği stratejiyi uygulayarak aynı şekilde oynamalıdır. Yani, Betül'ün hamlesinden sonra kalan taş sayısı da (1+5+1+5+...) şeklindeki toplamlardan birini oluşturacak şekilde olmalıdır.
Bu şekilde devam ettiklerinde, son taşı alan oyuncu oyunu kaybedecektir.
Örneğin:
Ahmet ilk hamlesinde 2018 = 1 + 5 + 1 + 5 + 1 + 5 + ... şeklinde bir toplam oluşturabilir. İlk turda 2018'den 1 taş alarak 2017 taş bırakır.
Betül, Ahmet'in stratejisine uygun olarak hareket ederse, 2017 = 5 + 1 + 5 + 1 + 5 + ... şeklinde bir toplam oluşturmalıdır. İkinci turda 2017'den 5 taş alarak 2012 taş bırakır.
Bu şekilde devam ettiklerinde, son taşı alan oyuncu kaybedecektir.

Sonuç olarak, Ahmet ilk hamlesinde yerden 1 taş alırsa, stratejisini doğru uygulayarak oyunu kazanmayı garantileyebilir.
